src/damavand/cloud/aws/resources/glue_component.py

```diff
@@ -8,21 +10,150 @@
 from pulumi import ResourceOptions
 
 
+class GlueWorkerType(Enum):
+    """Enum representing Glue worker types."""
+
+    G_1X = "G.1X"
+    G_2X = "G.2X"
+    G_4X = "G.4X"
+    G_8X = "G.8X"
+    G_025X = "G.025X"
+    Z_2X = "Z.2X"
+
+
+class GlueJobType(Enum):
+    """Enum representing Glue job types."""
+
+    GLUE_ETL = "glueetl"
+    GLUE_STREAMING = "gluestreaming"
```

Comment:
nitpick: The Enam already has glue in its name, better to remove it from the cases:

```suggestion
    ETL = "glueetl"
    STREAMING = "gluestreaming"
```

Reply:
Done